tests/nnstreamer_datarepo/unittest_datareposrc.cc

```diff
@@ -79,7 +78,7 @@ create_sparse_tensors_test_file ()
 
   const gchar *str_pipeline = g_strdup_printf (
       "datareposrc location=%s json=%s start-sample-index=0 stop-sample-index=9 ! "
-      "tensor_sparse_enc ! other/tensors,format=sparse,framerate=0/1 ! "
+      "tensor_sparse_enc ! other/tensors,format=sparse,framerate=30/1 ! "
```

Comment:
Let's keep testing ```0/1``` rate. Please add ```30/1``` test case seperately.

Reply:
Thank you. I will add new TC